qemu/tests/migration_multi_host.py

```diff
@@ -74,6 +75,15 @@ def migration_scenario(self, worker=None):
             def start_worker(mig_data):
                 logging.info("Try to login guest before migration test.")
                 vm = env.get_vm(params["main_vm"])
+
+                if "rdma" in self.mig_protocol:
+                    mig_speed = params.get("mig_speed", "40G")
+                    vm.monitor.migrate_set_speed(mig_speed)
+
+                    enable_rdma_pin_all = params.get("enable_rdma_pin_all",
+                                                     "migrate_set_capability rdma-pin-all on")
+                    vm.monitor.human_monitor_cmd(enable_rdma_pin_all)
```

Comment:
Hi @ypu, confirmed with feature owner, set_speed step is necessary for rdma migration.